src/libraries/sendtohelix.proj

```diff
@@ -79,9 +79,10 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetsMobile)' == 'true'">
-    <HelixPreCommands>"$HELIX_CORRELATION_PAYLOAD"/dotnet/dotnet tool install --tool-path "$HELIX_CORRELATION_PAYLOAD"/xharness-cli --version $(MicrosoftDotNetXHarnessTestsRunnersVersion) --add-source https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet-eng/nuget/v3/index.json Microsoft.DotNet.XHarness.CLI;</HelixPreCommands>
+    <HelixPreCommands>"$HELIX_CORRELATION_PAYLOAD"/dotnet/dotnet tool install --tool-path "$HELIX_CORRELATION_PAYLOAD"/xharness-cli --version $(MicrosoftDotNetXHarnessTestRunnersXunitVersion) --add-source https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet-eng/nuget/v3/index.json Microsoft.DotNet.XHarness.CLI;</HelixPreCommands>
```

Comment:
I don't think this would work as the correlation payload is just a runtime, not an SDK so I don't expect the `dotnet tool install` command to work. Just try using the globally installed SDK.

Reply:
This will actually work if you define following:
```xml
    <IncludeDotNetCli>true</IncludeDotNetCli>
    <DotNetCliPackageType>sdk</DotNetCliPackageType>
    <DotNetCliVersion>3.1.201</DotNetCliVersion>
```

In this PR, Jo did that in `sendtohelix.proj` above. I disagree with it because it will preinstall the SDK for every job using `sendtohelix` in `dotnet/runtime` which I don't think we want and we are discussing it above.

I agree that we could add support to Helix SDK to preinstall the XHarness tool like you suggest but the caveat is that the tool is built to run on 3.1.201 so we would need to make sure you don't set both XHarness version and a different sdk/runtime and version.

@MattGal do you think this is the right thing to do - as posted above - allow to specify `<XHarnessCliVersion>` as suggested in the comment above which would just prepare the XHarness CLI in the env and let people do whatever with it?